### PR TITLE
View products front end | Jack Chou

### PR DESCRIPTION
**Pull request by jackmchou**
image

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -18,8 +20,14 @@ export default class App extends React.Component {
...
 
   render() {
-    return this.state.isLoading
-      ? <h1>Testing connections...</h1>
-      : <h1>{ this.state.message }</h1>;
+    return (
+      <div>
+        {/* {this.state.isLoading
+          ? <h1>Testing connections...</h1>
+          : <h1>{this.state.message}</h1>} */}
```
Comment (thebearingedge) [inline client/components/app.jsx]: Please do not turn in commented out code.

File: client/components/product-list-item.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                    <p className="card-subtitle text-muted">$ {(product.price * 0.01).toFixed(2)}</p>
+                    <p className="card-text">{product.shortDescription}</p>
+                  </div>
+                </div>
+              </div>
+            );
+          })
+        }
+      </div>
+    </div>
```
Comment (thebearingedge) [inline client/components/product-list-item.jsx]: the mapping should happen in `ProductList`. `ProductListItem` should render one ... item.

File: client/components/app.jsx (hunk 1 above)
Comment (jackmchou) [reply client/components/app.jsx]: Fixed!

File: client/components/product-list-item.jsx (hunk 2 above)
Comment (jackmchou) [reply client/components/product-list-item.jsx]: Fixed!

**Pull request merged**